### DISCUSSION TITLE
Slinger error out

### DISCUSSION THREAD

**Discussion by Edgar-Wu** [GerryDazoo/Slinger#306] [Q&A]
Hi,

   I have followed the step-by-step instructions and copied the password in clear text from the log file but got the following when trying to view the stream from VLC.

  Any recommendations?

Thanks,

Edgar

*****************
Slinginfo  "Solo/Pro/ProHD" 12 30 192.168.1.141 5001 3000 10
8080 Starting Stream for  192.168.1.127:59392
Connecting... ('192.168.1.141', 5001) Control
cmd: 0x67 err: 0x2 8
8080 Error Starting Session. Check your admin password in config.ini file!
8080 ERROR: Slingbox session startup failed.
8080 Logging Out
EXCEPTION [WinError 10054] An existing connection was forcibly closed by the remote host 0x101 0 0x68 0 3 0 0x2000 0x0
Exception in thread Thread-2 (streamer):
Traceback (most recent call last):
  File "slingbox_server.py", line 372, in sling_cmd
    s_ctl.sendall( cmd )
ConnectionResetError: [WinError 10054] An existing connection was forcibly closed by the remote host

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "threading.py", line 1016, in _bootstrap_inner
  File "threading.py", line 953, in run
  File "slingbox_server.py", line 964, in streamer
    if s_ctl : s_ctl = closecontrol(s_ctl)
  File "slingbox_server.py", line 576, in closecontrol
    if Solo: sling_cmd( 0x68, b'')
  File "slingbox_server.py", line 375, in sling_cmd
    exit(1)
NameError: name 'exit' is not defined

**Comment by GerryDazoo** [comment]
There's two possibilities.
1) Your password really is incorrect or 
2) your Slingbox wasn't isolated from the Internet and has received the bad firmware update.

What model slingbox do you have? Would you mind posting the password you used? Fell free to x out a character or two.

PS. Please get the latest version of the code that error in the exception handling has been corrected.

**Reply by Edgar-Wu** [reply]
1. Password is: Unisys1$
2. Model is SB300/ProHD

Thank you.

**Reply by GerryDazoo** [reply]
OK then it looks like #2. There are many discussions here and how to try to revive a box.

**Reply by Edgar-Wu** [reply]
OK, I'll try to revive the box.
Thanks a lot, Gerry.
